modules/ding_ting_frontend/ding_ting_frontend.features.field_base.inc

```diff
@@ -321,7 +321,7 @@ function ding_ting_frontend_field_default_field_bases() {
     'field_name' => 'ting_type',
     'foreign keys' => array(),
     'indexes' => array(),
-    'locked' => 0,
+    'locked' => 1,
     'module' => 'ting',
     'settings' => array(),
     'storage' => array(
```

Comment:
These exports will lock the fields agian, that was unlocked in this issue: http://platform.dandigbib.org/issues/1445